src/bc.hpp

```diff
@@ -0,0 +1,123 @@
+#ifndef BC_HPP_
+#define BC_HPP_
+//========================================================================================
+// AthenaPK - a performance portable block structured AMR astrophysical MHD code.
+// Copyright (c) 2021, Athena-Parthenon Collaboration. All rights reserved.
+// Licensed under the 3-clause BSD License, see LICENSE file for details
+//========================================================================================
+//! \file bc.hpp
+//  \brief Custom boundary conditions for AthenaPK
+//
+// Computes reflecting boundary conditions using AthenaPK's cons variable pack.
```

Comment:
If this is only for reflecting boundary conditions, should this file be renamed? Should we have a `bc` folder?

Or maybe this file can keep this name as a place holder for other commonly used boundary functions in AthenaPK